src/tools/importPlanGoals.js

```diff
@@ -135,26 +100,50 @@ export default async function importGoals(fileKey, region) {
             const dbGrant = await Grant.findOne({ where: { ...fullGrant }, attributes: ['id', 'recipientId'] });
             if (!dbGrant) {
               // eslint-disable-next-line no-console
-              console.log(`Couldn't find grant: ${fullGrant.number}. Exiting...`);
+              logger.error(`Couldn't find grant: ${fullGrant.number}. Exiting...`);
               throw new Error('error');
             }
             const grantId = dbGrant.id;
-            await Goal.findOrCreate({
-              where: { grantId, name: goal.name, isFromSmartsheetTtaPlan: true },
-              defaults: goal,
+            const dbGoal = await Goal.findOne({
```

Comment:
What if we have multiple goals with the same grant and text but some are marked as complete? Do we need to take into account the goal status?

Reply:
Also assuming we don't care about this field anymore 'isFromSmartsheetTtaPlan'.

Reply:
> What if we have multiple goals with the same grant and text but some are marked as complete? Do we need to take into account the goal status?

Yes. The status will only be updated if it is newer.

Reply:
> Also assuming we don't care about this field anymore 'isFromSmartsheetTtaPlan'.

We still would want this for historical reasons.